src/eval-frame/components/reps/__test_helpers__/reps-test-value-cases.js

```diff
@@ -318,24 +318,39 @@ export const promiseCases = {
 // ==================== tabular
 
 export const rowTableCases = {
-  rowsTable_plainObjects: new Array(100).fill(0).map((x, i) => ({
+  rowsTable_plainObjects: new Array(58).fill(0).map((x, i) => ({
```

Comment:
changed a bunch of the text examples so they aren't all just 100 -- we should test more than just nice round numbers